Mathlib/Data/PFunctor/Multivariate/M.lean

```diff
@@ -143,7 +143,7 @@ set_option linter.uppercaseLean3 false in
 
 /-- Using corecursion, construct the contents of an M-type -/
 def M.corecContents {α : TypeVec.{u} n}
-                    {β : Type u}
+    {β : Type u}
```

Comment:
Same as above, could you either undo this or change the whole statement?

Reply:
done

Reply:
should be done now